win32ss/user/ntuser/focus.c

```diff
@@ -341,6 +341,11 @@ IntDeactivateWindow(PTHREADINFO pti, HANDLE tid)
       UserDerefObjectCo(pwndFocus);
    }
 
+   //
+   // Check for keyboard modifiers and release them (CORE-14768)
+   //
```

Comment:
```suggestion
   // Check for keyboard modifiers and release them (CORE-14768)
```
Yes, other code in here does already use the LOC-wasting-comment-style, but I am pretty sure that this will be subject of change in the future. We shouldn't add more work to that. In C code it is very handy not having to scroll so far, e.g. when dealing with resource-freeing it is helpful to have code fitting the screen vertically.

Reply:
OK.